sundog/views.py

```diff
@@ -1031,22 +839,14 @@ def debt_add_note(request):
             response_data = 'Ok'
             response = {'result': response_data}
         else:
-            form_errors = []
-            for field in form:
-                if field.errors:
-                    for field_error in field.errors:
-                        error = strip_tags(field.html_name.replace("_", " ").title()) + ": " + field_error
-                        form_errors.append(error)
-            for non_field_error in form.non_field_errors():
-                form_errors.append(non_field_error)
-            response = {'errors': form_errors}
+            response = {'errors': get_form_errors(form)}
         return JsonResponse(response)
 
 
+@login_required
 def add_enrollment_plan(request):
     form_errors = None
-    form_fee_1 = FeePlanForm(prefix='1')
-    form_fee_2 = FeePlanForm(prefix='2')
+    form_fee_1 = FeeForm(prefix='1')
```

Comment:
FeeForm is different from FeePlanForm.
Also why is the second fee form removed?